percona-xtradb-cluster/5.6/source/release-notes/Percona-XtraDB-Cluster-5.6.24-25.11.rst

```diff
@@ -0,0 +1,56 @@
+.. rn:: 5.6.24-25.11
+
+=======================================
+ |Percona XtraDB Cluster| 5.6.24-25.11 
+=======================================
+
+Percona is glad to announce the release of |Percona XtraDB Cluster| 5.6 on June 3rd 2015. Binaries are available from `downloads area <http://www.percona.com/downloads/Percona-XtraDB-Cluster-56/release-5.6.24-25.11/>`_ or from our :doc:`software repositories </installation>`.
+
+Based on `Percona Server 5.6.24-72.2 <http://www.percona.com/doc/percona-server/5.6/release-notes/Percona-Server-5.6.24-72.2.html>`_ including all the bug fixes in it, *Galera Replicator* `3.11 <https://github.com/codership/galera/milestones/25.3.11>`_ and on *Codership wsrep API* `25.11 <https://github.com/codership/mysql-wsrep/milestones/5.6.x-25.11>`_ is now the current **General Availability** release. All of |Percona|'s software is open-source and free, all the details of the release can be found in the `5.6.24-25.11 milestone <https://launchpad.net/percona-xtradb-cluster/+milestone/5.6.24-25.11>`_ at Launchpad.
+
+New Features
+============
+
+ |Percona XtraDB Cluster| now allows reads in non-primary state by introducing a new session variable, :variable:`wsrep_dirty_reads`. This variable is boolean and is ``OFF`` by default. When set to ``ON``, a |Percona XtraDB Cluster| node accepts queries that only read, but not modify data even if the node is in the non-PRIM state (:bug:`1407770`).
+
+ |Percona XtraDB Cluster| now allows queries against ``INFORMATION_SCHEMA`` and ``PERFORMANCE_SCHEMA`` even with variables :variable:`wsrep_ready` and :variable:`wsrep_dirty_reads` set to ``OFF`` (:bug:`1409618`).
+ 
+ :variable:`wsrep_replicate_myisam` variable is now a session variable (:bug:`1280280`).
+
+ |Percona XtraDB Cluster| now uses ``getifaddrs`` for node address detection (:bug:`1252700`). 
+
+ |Percona XtraDB Cluster| has implemented two new status variables: :variable:`wsrep_cert_bucket_count` and :variable:`wsrep_gcache_pool_size` for better instrumentation of galera memory usage. :variable:`wsrep_cert_bucket_count` shows the number of cells in the certification index hash-table and :variable:`wsrep_gcache_pool_size` shows the size of the page pool and/or dynamic memory allocated for gcache (in bytes).
+
+Bugs fixed 
+==========
+
+ Using concurrent ``REPLACE``, ``LOAD DATA REPLACE`` or ``INSERT ON DUPLICATE KEY UPDATE`` statements in the ``READ COMMITTED`` isolation level or with the :variable:`innodb_locks_unsafe_for_binlog` option enabled could lead to a unique-key constraint violation. Bug fixed :bug:`1308016`.
+
+ Using the Rolling Schema Upgrade as a schema upgrade method due to conflict with :variable:`wsrep_desync` would allows only one ``ALTER TABLE`` to run concurrently. Bugs fixed :bug:`1330944` and :bug:`1330941`.
+
+ SST would resume even when the donor was already detected as being in ``SYNCED`` state. Bug fixed :bug:`1288528`.
+
```

Comment:
I think some more details here is better.  Can you mention the conflict with wsrep_desync and it being OFF which caused the conflict and resumed the donor earlier.